lib/revision_table_access_check_filter.js

```diff
@@ -34,7 +34,13 @@ module.exports = function(hyper, req, next, options, specInfo) {
             }
             return hyper.get({
                 uri: new URI(htmlPath),
-                headers: req.headers
+                /* TODO FIXME TEMP
+                 * This removes the cache-control header when requesting the HTML
+                 * for redirect inspection. However, this is a temporary hack in
+                 * order to avoid re-renders for no-cache requests. We need to
+                 * implement a proper header inspection routine.
+                 */
+                headers: Object.assign({}, req.headers, { 'cache-control': undefined })
```

Comment:
I was wondering the same thing. Probably the only header we need is the cookie one, in case it's present, but given this was a hot fix, didn't want to poke the devil.